### DISCUSSION TITLE
A moose bit my sister once

### DISCUSSION THREAD

**Discussion by philzook58** [philzook58/philzook58.github.io#10] [General]
No realli!

**Comment by tekknolagi** [comment]
dam is she ok

**Reply by philzook58** [reply]
minus the bite, ye

**Comment by aviskhatiwada** [comment]
How'd it happen? I thought moose don't bite unless they feel intruded on?

**Reply by philzook58** [reply]
It was her fault. I told her to let the moose win a hand.
